src/data/meta.json

```diff
@@ -0,0 +1,37 @@
+{
+    "_comment": "remove the _ before a non-comment/info propertie to enable it",
+    "_comment2": "For more info use json schemas to get a description and allowed value for every properties",
+    "_comment3": "https://github.com/ManualForArchipelago/Manual/blob/main/docs/resources/json-schemas-for-world-devs.md",
```

Comment:
I feel like we should handle multi-line comments differently. Like having an array or something. Like apworld_description is doing.